doc/backend/branding.md

```diff
@@ -136,3 +136,89 @@ like `@page`. Supported css features depending on weasyprint, see [here](https:/
 
 The second one (`rich_text_preview.template`) is a templating file of html for pandoc, see [here](https://pandoc.org/MANUAL.html#template-syntax)
 for specific template syntax.
+
+
+## App Custom Action
+
+In content apps (File, Thread, Note, Kanban, Logbook), there is a kebab menu accessible by the button "⋮".
+
+App Custom Action allows to add submenu options with custom link.
+
+### Create an App Custom Actions
+
+Update the file `frontend/dist/assets/branding/app_custom_actions.json`.
+
+It is a json file containing a list of objects. Each objects are the definition of an App Custom Actions.
+
+**Available parameters:**
+- `icon_text`:
+  - The icon preceding the submenu option. Use it for icon from css library.
+  - Example: `"icon_text": "fas fa-wine-bottle"`
```

Comment:
`from css library`: which librairy available for this ? I think it is better to explain which library is used by default with tracim.

Reply:
will fix

Reply:
fixed